### ISSUE TITLE
aws-lambda-event-sources: Add new config feature for group id to SelfManagedKafkaEventSource

### ISSUE THREAD

**Issue by ghdoergeloh** [aws/aws-cdk#21734]
### Describe the feature

Can we add the new option to set the group id ("SelfManagedKafkaEventSourceConfig" now also supported by [Cloud Formation](https://docs.aws.amazon.com/de_de/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-eventsourcemapping.html#cfn-lambda-eventsourcemapping-selfmanagedkafkaeventsourceconfig)), to the [SelfManagedKafkaEventSource](https://github.com/aws/aws-cdk/blob/8a733bad022fb2ac0551e32fd26a4847c0b36ff2/packages/%40aws-cdk/aws-lambda-event-sources/lib/kafka.ts#L176)?

### Use Case

In our company, the group ID is used in ACLs to restrict access and for logging. A UUID is not very meaningful. It also has to be updated in the ACLs every time a change of the EventSourceMapping requires a redeployment.

### Proposed Solution

Add a new optional argument to the SelfManagedKafkaEventSource constructor:

```
const kafkaSource = new SelfManagedKafkaEventSource({
  topic: '...',
  bootstrapServers: ['...','...'],
  groupId: '...' // 👈
});
```

### Other Information

The new property might be too new. It is not yet there in `CfnEventSourceMappingProps`.

### Acknowledgements

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### CDK version used

2.38.1

### Environment details (OS name and version, etc.)

Linux Ubuntu 22.04.1

**Comment by RaphaelManke** [comment]
I created a PR for that which adds a `consumerGroupId` attribute to set this value to self managed and aws managed kafka cluster.

**Comment by kaizencc** [comment]
Seems to be closed by #21791
